meetup/README.md

```diff
@@ -36,31 +42,38 @@ your script and create an OAUTH consumer through Meetup.com.
     **Note: While you can choose your own values for "Consumer Name" and
     "Application Website", you must follow the pattern for the "Redirect URI" in
     the table above**.
+1.  Visit [Apps Script] and open your Meetup connector Apps Script project.
+    Click on **File** -> **Project Properties** -> **Script Properties**.
+    Using the information obtained from Meetup, add the following key-value pairs
+    as script properties:
+    Key                   | Value
```

Comment:
Looks like this table is broken.

Reply:
hmm, preview was ok but looks like I need to add a line. Done!